testreports/releases/Madagascar/README.md

```diff
@@ -0,0 +1,5 @@
+# Test Reports
+
```

Comment:
We can give the link to the test report. Later find a better way to provide
If you upload binary file for test reports, each fix, you need to reupload. Please check which is better

Reply:
Thanks, removed binary file and kept link

Reply:
Given link has SODA_Delfin_Lamu_Test_Report.xlsx - wrong file?!